cmd/admin-handlers-config-kv.go

```diff
@@ -201,6 +208,19 @@ func setConfigKV(ctx context.Context, objectAPI ObjectLayer, kvBytes []byte) (re
 		return
 	}
 
+	// Check if callhome getting enabled, if so set the subnet proxy value to logger_webhook's subnet proxy
+	// Do this only if logger_wenhook's subnet proxy is not yet set
+	callhomeCfg := result.Cfg[config.CallhomeSubSys]["_"]
+	if result.SubSys == config.CallhomeSubSys && callhomeCfg.Get(callhome.Enable) == "on" {
```

Comment:
The `callhome` subsystem is for configuring the daily diagnostics callhome, and not logs. So this condition is not required. Instead, we should check whether the `subnet` subsystem config is being modified.

Reply:
So you mean during `mc admin config set ALIAS/ subnet proxy=VALUE`, we should set logger webhook proxy as well and not while `mc admin config set ALIAS/ callhome enable=on`, right ?

Reply:
yes, that's right